Index: jenkins-dind/Dockerfile
===================================================================
--- jenkins-dind.orig/Dockerfile
+++ jenkins-dind/Dockerfile
@@ -58,11 +58,11 @@ RUN { \
 ENV JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64
 
 ENV JAVA_VERSION 8u121
-ENV JAVA_DEBIAN_VERSION 8u121-b13-1~bpo8+1
+ENV JAVA_DEBIAN_VERSION 8u121-b13-0ubuntu1.16.04.2
 
 # see https://bugs.debian.org/775775
 # and https://github.com/docker-library/java/issues/19#issuecomment-70546872
-ENV CA_CERTIFICATES_JAVA_VERSION 20161107~bpo8+1
+ENV CA_CERTIFICATES_JAVA_VERSION 20160321
 
 RUN set -x \
 	&& apt-get update \
