Index: jenkins-dind/Dockerfile
===================================================================
--- jenkins-dind.orig/Dockerfile
+++ jenkins-dind/Dockerfile
@@ -70,11 +70,11 @@ RUN ln -svT "/usr/lib/jvm/java-8-openjdk
 ENV JAVA_HOME /docker-java-home
 
 ENV JAVA_VERSION 8u171
-ENV JAVA_DEBIAN_VERSION 8u171-b11-1~deb9u1
+ENV JAVA_DEBIAN_VERSION 8u171-b11-0ubuntu0.18.04.1
 
 # see https://bugs.debian.org/775775
 # and https://github.com/docker-library/java/issues/19#issuecomment-70546872
-ENV CA_CERTIFICATES_JAVA_VERSION 20170531+nmu1
+ENV CA_CERTIFICATES_JAVA_VERSION 20170930ubuntu1
 
 RUN set -ex; \
 	\
